ui/media/js/main.js

```diff
@@ -777,7 +785,9 @@ function getCurrentUserRequest() {
             stream_progress_updates: true,
             stream_image_progress: (numOutputsTotal > 50 ? false : streamImageProgressField.checked),
             show_only_filtered_image: showOnlyFilteredImageField.checked,
-            output_format: outputFormatField.value
+            output_format: outputFormatField.value,
+            use_face_correction: useFaceCorrectionField.checked,
```

Comment:
Why are these lines being added here, if they're already being added in line 810?

Reply:
Because the existing code at line 810 only works when the boxes are checked, but we also want to actively capture the unchecked state to be able to restore it (otherwise it will just be skipped by TASK_MAPPING). On top of that, I hadn't noticed this code was even there in the first place (I thought these two were just forgotten and added it).

Now that you brought it to my attention, I realized it's actually worse than that- this code is trying to do two things at the same time: capture the state of the checkbox *and* the face correction/upscaling method. I removed the 810 code and refined my addition to better follow the initial implementation's intent while also addressing the box unchecked scenario:
            use_face_correction: useFaceCorrectionField.checked,
            face_correction_method: 'GFPGANv1.3', // hardcoded for now
            use_upscale: useUpscalingField.checked,
            upscale_model: upscaleModelField.value,

Moreover, the init_image and save_to_disk_path calculations just above suffer from the very same issue and currently won't restore the unchecked state because they're just ignoring it.

If we're aligned on the overall direction, I can fix these too while I'm at it.

Thoughts?